Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -16,6 +16,7 @@
 
 import FirebaseFirestore
 import FirebaseSharedSwift
+@implementationOnly import FirebaseCoreExtension
```

Comment:
I think it might be `@_implementationOnly`

Reply:
Yep– thanks. Resolved and fixed the changelog in the latest commits I just pushed.